specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2021-01-01-preview/examples/NameSpaces/SBNameSpaceCreate.json

```diff
@@ -0,0 +1,141 @@
+{
+  "parameters": {
+    "namespaceName": "NamespaceSample",
+    "resourceGroupName": "ResurceGroupSample",
+    "api-version": "2021-01-01-preview",
```

Comment:
@v-Ajnava - For quicker API review, can you please follow below two?

1. Update the change log and other items in the description providing details on what all changed in the PR.
2. To easily identify the delta changes from previous api version, the recommendation is to copy the existing version into new directory structure for first commit and then push new changes in subsequent commits. 

Please raise a new PR with the above recommendations for faster review.